sympy/solvers/solveset.py

```diff
@@ -1398,26 +1398,26 @@ def _transolve(f, symbol, domain):
 
     * Philosophy behind the module
 
-    The purpose of `\_transolve` is to take equations which are not
+    The purpose of ``transolve`` is to take equations which are not
     already polynomial in their generator(s) and to either recast them
     as such through a valid transformation or to solve them outright.
     A pair of helper functions for each class of supported
     transcendental functions are employed for this purpose. One
     identifies the transcendental form of an equation and the other
     either solves it or recasts it into a tractable form that can be
-    solved by  `solveset`.
+    solved by  ``solveset``.
     For example, an equation in the form `a*b**f(x) - c*d**g(x) = 0`
     can be transformed to
     `log(a) + f(x)*log(b) - log(c) - g(x)*log(d) = 0`
```

Comment:
While you are fixing this, this math should use LaTeX notation (there are also some other instances elsewhere in the file). Things like `*` and `**` do not render well in LaTeX, and it should use `\log`, etc.